docs/APIClient.md

```diff
@@ -150,6 +150,13 @@ data = sketch.explore('google.com', as_pandas=True)
 
 This will return back a pandas DataFrame with the search results.
 
+Another example search query:
+```
+data = sketch.explore('192.168.0.1 AND NOT timestamp_desc:foobar', as_pandas=True,return_fields='datetime,message,timestamp_desc')
```

Comment:
can you make this

```
data = sketch.explore(
    '192.168.......',
    as_pandas=True,
    return_fields='.....'
)
```

just to make it easier to read

Reply:
sure thing - done

and renamed the PR